index.html

```diff
@@ -146,59 +150,67 @@ <h1 class="visually-hidden">Преимущества Глэйси.</h1>
         </ul>
       </section>
 
-      <article>
-        <p>Новое в нашем блоге</p>
-        <h2><a href="">10 способов сервировки фруктовых щербетов к столу</a></h2>
+      <article class="blog">
+        <p class="blog-subject">Новое в нашем блоге</p>
+        <a class="blog-title" href=""><h2>10 способов сервировки фруктовых щербетов к столу</h2></a>
```

Comment:
Здесь лучше изменить разметку (первый вариант был правильным):
```suggestion
        <h2><a class="blog-title" href="">10 способов сервировки фруктовых щербетов к столу</a></h2>
```
`h2` по умолчанию блочный элемент и он растянет всю ссылку на ширину родительского блока. Область без текста станет тоже кликабельной.